packages/datastore/src/datastore.ts

```diff
@@ -209,42 +344,76 @@ namespace Datastore {
   };
 
   /**
-   * @internal
+   * An object representing a datastore transaction.
    */
   export
-  interface IContext {
-    /**
-     *
-     */
-    readonly inTransaction: boolean;
+  type Transaction = {
 
     /**
-     *
+     * The id of the transaction.
      */
-    readonly version: number;
```

Comment:
The version is needed, so that all peers can update their version based on all other peers. See the comment above about taking the max of a version when applying a remote transaction.